llvm/lib/Target/EraVM/EraVMISelLowering.cpp

```diff
@@ -1029,6 +1045,38 @@ SDValue EraVMTargetLowering::LowerBR_JT(SDValue Op, SelectionDAG &DAG) const {
   return SDValue(Res, 0);
 }
 
+SDValue EraVMTargetLowering::LowerSELECT(SDValue Op, SelectionDAG &DAG) const {
+  // try to fold select with u{add|sub|mul}.with.overflow
```

Comment:
Could we document the transformation better?

Reply:
The document is improved now.